sbt-mode.el

```diff
@@ -31,6 +31,25 @@
 
 (defvar sbt:command-history-temp nil)
 
+(defvar sbt:display-buffer-action nil
+  "Customizable ACTION parameter for `display-buffer', which see.
```

Comment:
The documentation could be made a bit more clear. Start by describing when this action is called and then you can continue with how it can be used. Remove the part about why it is necessary (you can move that to the implementation comment).